llvm/test/Object/Inputs/COFF/mips.yaml

```diff
@@ -0,0 +1,76 @@
+!COFF
+header: !Header
```

Comment:
Let's minimise this input file and inline it into the test that needs it. I think the only significant parts of it are the machine type and relocations. Most of the rest can be reduced/removed. It doesn't need to be a fully runnable binary, for instance, for the purposes of this feature.